src/pages/home/report/ReportActionItem.js

```diff
@@ -279,6 +288,11 @@ function ReportActionItem(props) {
         return (
             <>
                 {children}
+                {props.action.linkMetadata && (
+                    <LinkPreviewer
+                        linkMetadata={props.action.linkMetadata}
```

Comment:
I believe there should be just one link preview shown if mulitple links is in the message based on the PR body 
```suggestion
                        linkMetadata={props.action.linkMetadata}
                        maxAmountOfPreviews=1
```
<img width="956" alt="image" src="https://github.com/Expensify/App/assets/36083550/b478e039-83e7-44c1-948c-5453543ed7ca">